one_fm/one_fm/custom/leave_application.json

```diff
@@ -86,8 +86,8 @@
    "dt": "Leave Application",
    "fetch_from": null,
    "fetch_if_empty": 0,
-   "fieldname": "custom_propose_to_date",
-   "fieldtype": "Date",
```

Comment:
It seems propose to and from dates are removed from the custom fields

Reply:
I have resolved this, just that the position the field I added takes is the position where custom_propose_to_date is before. The field wasn't removed